packages/office-ui-fabric-react/src/components/Checkbox/Checkbox.scss

```diff
@@ -6,220 +6,239 @@
 // Office UI Fabric
 // --------------------------------------------------
 // Checkbox styles
-
-$ms-checkbox-label-size: 20px;
+$ms-checkbox-label-size: 20px !default;
 $ms-checkbox-transition-duration: 200ms;
 $ms-checkbox-transition-timing: cubic-bezier(.4, 0, .23, 1);
-$ms-checkbox-highContrast-iconBoxSize: 16px;
-
-@mixin highContrast-border($active-color, $bw-color, $size: 2px) {
-  @media screen and (-ms-high-contrast: active) {
-    border: $size solid $active-color;
-  }
-
-  @media screen and (-ms-high-contrast: black-on-white) {
-    border: $size solid $bw-color;
-  }
-}
-
-@mixin highContrast-color($active-color, $bw-color) {
-  @media screen and (-ms-high-contrast: active) {
-    color: $active-color;
-  }
-
-  @media screen and (-ms-high-contrast: black-on-white) {
-    color: $bw-color;
-  }
-}
+$ms-checkbox-highContrast-iconBoxSize: 16px !default;
 
+// Default state
 .root {
-  box-sizing: border-box;
-  color: $ms-color-neutralPrimary;
-  font-size: $ms-font-size-m;
-  font-weight: $ms-font-weight-regular;
-  min-height: 36px;
-  position: relative;
-}
-
-.textLabel {
-  font-size: $ms-font-size-m;
-  @include padding(0, 0, 0, 26px);
-  display: inline-block;
-  font-size: $ms-font-size-m;
-}
-
-// The hidden input
-.input {
-  position: absolute;
-  opacity: 0;
-  top: 8px
-}
-
-// The checkbox square container
-.label::before {
-  content: '';
-  display: inline-block;
-  border: 1px solid $ms-color-neutralTertiary;
-  width: $ms-checkbox-label-size;
-  height: $ms-checkbox-label-size;
-  font-weight: normal;
-  position: absolute;
-  box-sizing: border-box;
-  transition-property: background, border, border-color;
-  transition-duration: $ms-checkbox-transition-duration;
-  transition-timing-function: $ms-checkbox-transition-timing;
-}
-
-// The check mark icon
-.label::after {
-  @include ms-Icon--CheckMark;
-  font-family: 'FabricMDL2Icons';
-  display: none;
-  position: absolute;
-  font-weight: 900;
-  background-color: transparent;
-  font-size: $ms-font-size-s-plus;
-  top: 0;
-  color: $ms-color-white;
-  line-height: $ms-checkbox-label-size;
-  width: $ms-checkbox-label-size;
-  text-align: center;
-}
-
-// The checkbox label
-.label {
-  display: inline-block;
-  cursor: pointer;
-  margin-top: 8px;
```

Comment:
@mikewheaton This is what @phkuo was talking about. I removed the top margin to make it easier to center when using a single one. He said that it should likely cause a big version change; though this whole change should since it rewrites the whole dom and anyone overriding the css will be broken.

Anyway, just FYI that this will break people's implementations and we should version against that.